<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QmitkNiftySegViewControls</class>
 <widget class="QWidget" name="QmitkNiftySegViewControls">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>502</width>
    <height>700</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>NiftySeg</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_13">
     <item>
      <widget class="QLabel" name="m_ReferenceImageLabel">
       <property name="text">
        <string>Reference Image </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="m_ReferenceImageComboBox"/>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QTabWidget" name="m_NiftySegTabWidget">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>EM</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <layout class="QVBoxLayout" name="verticalLayout_8">
         <item>
          <widget class="QLabel" name="m_EMSegResearchPaperLabel">
           <property name="text">
            <string>EM Segmentation: Link to Research Paper</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLabel" name="m_EMMaskImageLabel">
             <property name="text">
              <string>Mask Image</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="m_EMMaskImageComboBox"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLabel" name="m_EMClassesLabel">
             <property name="text">
              <string>Classes</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="m_EMClassesIntegerSpinBox">
             <property name="minimum">
              <number>1</number>
             </property>
             <property name="maximum">
              <number>5</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QToolBox" name="m_EMToolBox">
           <property name="currentIndex">
            <number>1</number>
           </property>
           <widget class="QWidget" name="Initialisation">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>458</width>
              <height>394</height>
             </rect>
            </property>
            <attribute name="label">
             <string>Initialisation</string>
            </attribute>
            <layout class="QVBoxLayout" name="verticalLayout">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_3">
               <item>
                <widget class="QRadioButton" name="m_EMPriorsRadioButton">
                 <property name="text">
                  <string>priors</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QRadioButton" name="m_EMMeansRadioButton">
                 <property name="text">
                  <string>means</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QRadioButton" name="m_EMAutomaticRadioButton">
                 <property name="text">
                  <string>automatic</string>
                 </property>
                 <property name="checked">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_4">
               <item>
                <widget class="QComboBox" name="m_EMPriorsImagesComboBox"/>
               </item>
               <item>
                <widget class="QPushButton" name="m_EMPriorsImagesBrowsePushButton">
                 <property name="text">
                  <string>browse</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <widget class="QTextEdit" name="m_EMMeansTextEdit"/>
             </item>
             <item>
              <spacer name="verticalSpacer">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>408</width>
                 <height>169</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="MRF">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>458</width>
              <height>394</height>
             </rect>
            </property>
            <attribute name="label">
             <string>MRF</string>
            </attribute>
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_5">
               <item>
                <widget class="QRadioButton" name="m_EMMRFOnRadioButton">
                 <property name="layoutDirection">
                  <enum>Qt::RightToLeft</enum>
                 </property>
                 <property name="text">
                  <string>on</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QRadioButton" name="m_EMMRFOffRadioButton">
                 <property name="text">
                  <string>off</string>
                 </property>
                 <property name="checked">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_6">
               <item>
                <widget class="QLabel" name="m_EMProgressiveUpdateLabel">
                 <property name="text">
                  <string>progressive update</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QCheckBox" name="m_EMProgressiveUpdateCheckBox">
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_7">
               <item>
                <widget class="QLabel" name="m_EMStrengthLabel">
                 <property name="text">
                  <string>strength</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QDoubleSpinBox" name="m_EMStrengthDoubleSpinBox">
                 <property name="layoutDirection">
                  <enum>Qt::RightToLeft</enum>
                 </property>
                 <property name="maximum">
                  <double>4.000000000000000</double>
                 </property>
                 <property name="singleStep">
                  <double>0.100000000000000</double>
                 </property>
                 <property name="value">
                  <double>0.500000000000000</double>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <spacer name="verticalSpacer_2">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>428</width>
                 <height>299</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="biasField">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>458</width>
              <height>394</height>
             </rect>
            </property>
            <attribute name="label">
             <string>bias field</string>
            </attribute>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_8">
               <item>
                <widget class="QRadioButton" name="m_EMBiasFieldOnRadioButton">
                 <property name="layoutDirection">
                  <enum>Qt::RightToLeft</enum>
                 </property>
                 <property name="text">
                  <string>on</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QRadioButton" name="m_EMBiasFieldOffRadioButton">
                 <property name="text">
                  <string>off</string>
                 </property>
                 <property name="checked">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_9">
               <item>
                <widget class="QLabel" name="m_EMPolynomialOrderLabel">
                 <property name="text">
                  <string>Order of Polynomials</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QSpinBox" name="m_EMPolynomialOrderIntegerSpinBox">
                 <property name="maximum">
                  <number>6</number>
                 </property>
                 <property name="value">
                  <number>4</number>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <spacer name="verticalSpacer_3">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>428</width>
                 <height>320</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="priorRelaxation">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>458</width>
              <height>394</height>
             </rect>
            </property>
            <attribute name="label">
             <string>prior relaxation</string>
            </attribute>
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_10">
               <item>
                <widget class="QRadioButton" name="m_EMPriorRelaxationOnRadioButton">
                 <property name="layoutDirection">
                  <enum>Qt::RightToLeft</enum>
                 </property>
                 <property name="text">
                  <string>on</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QRadioButton" name="m_EMPriorRelaxationOffRadioButton">
                 <property name="text">
                  <string>off</string>
                 </property>
                 <property name="checked">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_12">
               <item>
                <widget class="QLabel" name="m_EMWeightLabel">
                 <property name="text">
                  <string>weight</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QDoubleSpinBox" name="m_EMWeightDoubleSpinBox">
                 <property name="maximum">
                  <double>1.000000000000000</double>
                 </property>
                 <property name="singleStep">
                  <double>0.100000000000000</double>
                 </property>
                 <property name="value">
                  <double>0.500000000000000</double>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_11">
               <item>
                <widget class="QLabel" name="m_EMKernelSizeLabel">
                 <property name="text">
                  <string>kernel size</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QDoubleSpinBox" name="m_EMKernelSizeDoubleSpinBox">
                 <property name="maximum">
                  <double>99999.990000000005239</double>
                 </property>
                 <property name="value">
                  <double>3.000000000000000</double>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <spacer name="verticalSpacer_4">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>418</width>
                 <height>292</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Load</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <layout class="QVBoxLayout" name="verticalLayout_9">
         <item>
          <widget class="QLabel" name="m_LoadSegResearchPaperLabel">
           <property name="text">
            <string>Load Segmentation: Link to Research Paper</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_14">
           <item>
            <widget class="QLabel" name="m_LoadMaskImageLabel">
             <property name="text">
              <string>Mask Image</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="m_LoadMaskImageComboBox"/>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QToolBox" name="m_LoadToolBox">
           <property name="currentIndex">
            <number>2</number>
           </property>
           <widget class="QWidget" name="MRF_3">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>458</width>
              <height>449</height>
             </rect>
            </property>
            <attribute name="label">
             <string>MRF</string>
            </attribute>
            <layout class="QVBoxLayout" name="verticalLayout_5">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_15">
               <item>
                <widget class="QRadioButton" name="m_LoadMRFOnRadioButton">
                 <property name="layoutDirection">
                  <enum>Qt::RightToLeft</enum>
                 </property>
                 <property name="text">
                  <string>on</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QRadioButton" name="m_LoadMRFOffRadioButton">
                 <property name="text">
                  <string>off</string>
                 </property>
                 <property name="checked">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_16">
               <item>
                <widget class="QLabel" name="m_LoadProgressiveUpdateLabel">
                 <property name="text">
                  <string>progressive update</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QCheckBox" name="m_LoadProgressiveUpdateCheckBox">
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_17">
               <item>
                <widget class="QLabel" name="m_LoadStrengthLabel">
                 <property name="text">
                  <string>strength</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QDoubleSpinBox" name="m_LoadStrengthDoubleSpinBox">
                 <property name="layoutDirection">
                  <enum>Qt::RightToLeft</enum>
                 </property>
                 <property name="maximum">
                  <double>4.000000000000000</double>
                 </property>
                 <property name="singleStep">
                  <double>0.100000000000000</double>
                 </property>
                 <property name="value">
                  <double>0.500000000000000</double>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <spacer name="verticalSpacer_5">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>418</width>
                 <height>354</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="biasField_3">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>458</width>
              <height>449</height>
             </rect>
            </property>
            <attribute name="label">
             <string>bias field</string>
            </attribute>
            <layout class="QVBoxLayout" name="verticalLayout_7">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_18">
               <item>
                <widget class="QRadioButton" name="m_LoadBiasFieldOnRadioButton">
                 <property name="layoutDirection">
                  <enum>Qt::RightToLeft</enum>
                 </property>
                 <property name="text">
                  <string>on</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QRadioButton" name="m_LoadBiasFieldOffRadioButton">
                 <property name="text">
                  <string>off</string>
                 </property>
                 <property name="checked">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_19">
               <item>
                <widget class="QLabel" name="m_LoadPolynomialOrderLabel">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="text">
                  <string>Order of Polynomials</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QSpinBox" name="m_LoadPolynomialOrderIntegerSpinBox">
                 <property name="maximum">
                  <number>6</number>
                 </property>
                 <property name="value">
                  <number>4</number>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <spacer name="verticalSpacer_6">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>408</width>
                 <height>375</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="priorRelaxation_3">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>458</width>
              <height>449</height>
             </rect>
            </property>
            <attribute name="label">
             <string>prior relaxation</string>
            </attribute>
            <layout class="QVBoxLayout" name="verticalLayout_6">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_20">
               <item>
                <widget class="QRadioButton" name="m_LoadPriorRelaxationOnRadioButton">
                 <property name="layoutDirection">
                  <enum>Qt::RightToLeft</enum>
                 </property>
                 <property name="text">
                  <string>on</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QRadioButton" name="m_LoadPriorRelaxationOffRadioButton">
                 <property name="text">
                  <string>off</string>
                 </property>
                 <property name="checked">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_22">
               <item>
                <widget class="QLabel" name="m_LoadWeightLabel">
                 <property name="text">
                  <string>weight</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QDoubleSpinBox" name="m_LoadWeightDoubleSpinBox">
                 <property name="maximum">
                  <double>1.000000000000000</double>
                 </property>
                 <property name="singleStep">
                  <double>0.100000000000000</double>
                 </property>
                 <property name="value">
                  <double>0.500000000000000</double>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_21">
               <item>
                <widget class="QLabel" name="m_LoadKernelSizeLabel">
                 <property name="text">
                  <string>kernel size</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QDoubleSpinBox" name="m_LoadKernelSizeDoubleSpinBox">
                 <property name="maximum">
                  <double>99999.990000000005239</double>
                 </property>
                 <property name="value">
                  <double>3.000000000000000</double>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <spacer name="verticalSpacer_7">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>428</width>
                 <height>347</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="m_ExecuteSaveCancelPushButtonsHLayout">
     <item>
      <widget class="QPushButton" name="m_NiftySegCancelPushButton">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="m_NiftySegSaveAsPushButton">
       <property name="text">
        <string>Save As</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="m_NiftySegExecutePushButton">
       <property name="text">
        <string>Execute</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
